File: Sep3-SemesterProject/Tier1-Presentation/Domain/DTOs/AppointmentResolveDto.cs
namespace Domain.DTOs;

public class AppointmentResolveDto
{
    public int Id { get; }
    public string? Diagnosis { get; set; }
    public bool? Status { get; set; }

    public AppointmentResolveDto(int id)
    {
        Id = id;
    }
}

File: Sep3-SemesterProject/Tier2-BusinessLogic/src/main/java/via/sdj3/proofofconcept_v3/Dto/AppointmentResolveDto.java
package via.sdj3.proofofconcept_v3.Dto;

public class AppointmentResolveDto {
    private final int id;
    private String diagnosis;
    private Boolean status;

    public AppointmentResolveDto(int id) {
        this.id = id;
    }

    public int getId() {
        return id;
    }

    public String getDiagnosis() {
        return diagnosis;
    }

    public void setDiagnosis(String diagnosis) {
        this.diagnosis = diagnosis;
    }

    public Boolean getStatus() {
        return status;
    }

    public void setStatus(Boolean status) {
        this.status = status;
    }
}